pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -27,6 +27,17 @@ def test_roundtrip_pickle_with_tz(self):
         unpickled = tm.round_trip_pickle(index)
         tm.assert_index_equal(index, unpickled)
 
+    @pytest.mark.parametrize("dtype", [
+        None, 'datetime64[ns, CET]',
+        'datetime64[ns, EST]', 'datetime64[ns, UTC]'
+    ])
+    def test_date_accessor(self, dtype):
+        # GH 21230
+
+        index = DatetimeIndex(['2013-01-24 15:01:00'], dtype=dtype)
+        tm.assert_numpy_array_equal(index.date,
```

Comment:
use 
```
result = 
expected = 
```

Reply:
can you add another date that is not ambiguous (e.g. its on the same date regardless of tz)
can you add a NaT as well